libsignal-service-actix/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 
 [dependencies]
 libsignal-service = { path = "../libsignal-service" }
-libsignal-protocol = { git = "https://github.com/Michael-F-Bryan/libsignal-protocol-rs" }
+libsignal-protocol = { path = "../../libsignal-protocol-rs/libsignal-protocol" }
```

Comment:
This line and the corresponding line below need to be reverted to be able to compile.